docs/bigcommerce-for-wordpress/extending-the-plugin/customization-guide.md

```diff
@@ -289,20 +385,21 @@ function myCheckoutFunction($checkout_config) {
   $checkout_config['styles']['step']['color'] = '#0BE640'; //green
   return $checkout_config;
 }
-add_filter( 'bigcommerce/checkout/config', 'myCheckoutFunction');
-
+add_filter('bigcommerce/checkout/config', 'myCheckoutFunction');
 ```
 
 Following this format, you can apply styles to other elements, like buttons, input fields, and checkboxes. See the full list of checkout elements that can be styled and which properties you can adjust in the [Embedded Checkout Styles documentation](https://github.com/bigcommerce/checkout-sdk-js/blob/master/docs/interfaces/embeddedcheckoutstyles.md) (Github).
 
 Note that styles apply globally to all elements on the checkout page. For example, styles applied to steps will apply to all steps rather than targeting only step 2 or 3.
 
 ## Email Templates
-You may wish to customize the built-in transactional emails sent from BigCommerce when an order is placed or updated. You can add custom text or images to email templates to reflect your store’s branding. 
+
+You may wish to customize the built-in transactional emails sent from BigCommerce when an order is placed or updated. You can add custom text or images to email templates to reflect your store’s branding.

```

Comment:
You may wish to customize the built-in transactional emails sent from BigCommerce when placing or updating an order. You can add custom text or images to email templates to reflect your store’s branding.